sdk/keyvault/azure-security-keyvault-administration/src/private/keyvault_settings_common_request.hpp

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Provides a wrapper class for the Azure Core Pipeline for all Key Vault services where
+ * common functionality is set up.
+ *
+ */
+
+#pragma once
+
+#include <azure/core/context.hpp>
+#include <azure/core/http/http.hpp>
+#include <azure/core/internal/http/pipeline.hpp>
+#include <azure/core/internal/json/json.hpp>
+#include <azure/core/internal/json/json_serializable.hpp>
+#include <azure/core/response.hpp>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace _detail {
```

Comment:
Why is this at the `KeyVault::_detail` level and not at the `KeyVault::Administration::_detail` level?